src/safeds/ml/nn/layers/_forward_layer.py

```diff
@@ -21,18 +21,25 @@ class ForwardLayer(Layer):
     neuron_count:
         The number of neurons in this layer
 
+    activation_function:
+        The activation function used in the forward layer
+
     Raises
     ------
     OutOfBoundsError
         If input_size < 1
         If output_size < 1
+
+    ValueError
+        If the given activation function does not exist
     """
 
-    def __init__(self, neuron_count: int):
+    def __init__(self, neuron_count: int, activation_function: str = "notset"):
```

Comment:
If I understand correctly, if this variable is anything other than `"notset"` the default activation function will be overwritten. Therefore, I would call the parameter something like `overwrite_activation_function` to clarify this behavior. Similar to this, I would clarify this behavior more clear in the doc string. Also, please do not use `str` but `Literal` to specify the allowed activation functions.

Reply:
added